openforcefield/topology/topology.py

```diff
@@ -1547,20 +1547,86 @@ def from_openmm(cls, openmm_topology, unique_molecules=None):
         # TODO: How can we preserve metadata from the openMM topology when creating the OFF topology?
         return topology
 
-    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API.
-    #       Given the recent (2019_04) discussions about potential loss of parameters during conversion, we should
-    #       revisit this function to determine what sorts of guarantees we can put on system correctness before
-    #       we expose it.
-    def _to_openmm(self):
+    def to_openmm(self):
         """
         Create an OpenMM Topology object.
 
+        The OpenMM ``Topology`` object will have one residue per topology
+        molecule. Currently, the number of chains depends on how many copies
+        of the same molecule are in the ``Topology``. Molecules with more
+        than 5 copies are all assigned to a single chain, otherwise one
+        chain is created for each molecule. This behavior may change in
```

Comment:
This 5-copy cutoff is a good idea. We can discuss it more, but should put an unstable-API warning here for now.